coverage/.resultset.json

```diff
@@ -0,0 +1,110 @@
+{
+  "RSpec": {
+    "coverage": {
+      "/Users/maryballantyne/Turing/enigma_project/lib/enigma.rb": {
```

Comment:
again want to explore the use of .gitignore to eliminate extra data being pulled into main branch